molecule/default/tests/test_docker.py

```diff
@@ -0,0 +1,40 @@
+import os
+
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('docker-docker')
```

Comment:
should be `get_hosts('docker'`)`, at the moment tests are being skipped:
```
collected 7 items                                                              
    
    tests/test_docker-inactive.py .                                          [ 14%]
    tests/test_docker.py ssssss                                              [100%]
    
    ===================== 1 passed, 6 skipped in 1.41 seconds ======================
```

Reply:
Thanks for checking, 920dfbc should re-enable the tests